scripts/generate-secret.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+if [[ -z $DO_ACCESS_TOKEN ]]; then
+  echo "DO_ACCESS_TOKEN is empty, this must be set to your DigitalOcean access token..."
+  exit 1
+fi
+
+ENCODED_ACCESS_TOKEN=$(echo -n "$DO_ACCESS_TOKEN" | base64)
+
+GENERATED_SECRET=./generated-secret.yml
+trap "{ rm $GENERATED_SECRET; }" EXIT
```

Comment:
Maybe we should leave the file, so user can copy it or choose the another cluster (or maybe even better ask user does it want to keep the file).

Reply:
The secret is outputted as part of the script, so they can just copy the stdout from the script. Thoughts?

Reply:
Oh, my bad -- I thought it's returning only the secret, not the whole file. In that case, it's okay and should stay as it is.